lib/devices/sensor.js

```diff
@@ -85,6 +85,16 @@ module.exports = function(device, api, capabilities) {
       });
   }
 
+  // PM2.5 Sensor
+  if ('measure_pm25' in capabilities) {
+    homekitAccessory
+      .addService(Service.AirQualitySensor)
```

Comment:
Is there a reason why you don't pass `device.name` as second argument like with the rest of the services?